### PR TITLE
Refactor cog names

### PR DESCRIPTION
**Pull request by MattyTheHacker**
This brings the induct command in line with #232 when that's merged

### CODE REVIEW

**Review by CarrotManMatt — APPROVED** [review]
Command cogs only need to specify that they are a slash command cog if they are disambiguating themself from the user/message-context-commands version of that cog. I.e. only the induct cogs need renaming.

**Review by CarrotManMatt — CHANGES_REQUESTED** [review]
Wrong button. I'm competent reviewer.

File: cogs/induct.py [hunk 1]
```diff
@@ -356,7 +356,7 @@ async def induct(self, ctx: TeXBotApplicationContext, str_induct_member_id: str,
         await self._perform_induction(ctx, induct_member, silent=silent)
 
 
-class InductUserCommandsCog(BaseInductCog):
+class InductContextCommandsCog(BaseInductCog):
     """Cog class that defines the context menu induction commands & their call-back methods."""
```
Comment (CarrotManMatt) [inline cogs/induct.py:360]: Term "context-menu" should be hyphenated.

**Review by CarrotManMatt — CHANGES_REQUESTED** [review]

File: cogs/induct.py (hunk 1 above)
Comment (MattyTheHacker) [reply cogs/induct.py:360]: This issue was not introduced in this pr and as it is completely unrelated to the purpose of the pr are out of scope

**Review by CarrotManMatt — APPROVED** [review]

**Comment by CarrotManMatt** [comment]
Look what you've done now. Now we need someone else to review.

**Comment by MattyTheHacker** [comment]
Nah revert that shit rn no cap bruv

**Comment by MattyTheHacker** [comment]
@CarrotManMatt go on then smh

**Review by CarrotManMatt — APPROVED** [review]

**Pull request merged**